### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by MathiasReker**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Fix text
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | https://github.com/PrestaShop/PrestaShop/issues/10534
| How to test?  | Go to usefull links in BO

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/10535)

### CODE REVIEW

**Review by PierreRambaud — DISMISSED** [review]

**Comment by PierreRambaud** [comment]
@colinegin @eternoendless What do you think?

**Comment by khouloudbelguith** [comment]
Fixed by https://github.com/PrestaShop/PrestaShop/pull/10151

**Comment by MathiasReker** [comment]
#10535 fix typo
#10151 fix url

**Comment by colinegin** [comment]
@eternoendless @LouiseBonnard didn't we say we keep the generic word "Bug Tracker" in order to avoid wording changes if we use new tools in the future ?

**Comment by LouiseBonnard** [comment]
@eternoendless, @colinegin, what about _Report issues_? Without any further details, we will be free to use new tools in the future without adapting the wording.

File: app/Resources/translations/default/AdminDashboardFeature.xlf [hunk 1]
```diff
@@ -109,14 +109,14 @@ File: admin-dev/themes/default/template/controllers/dashboard/helpers/view/view.
...
+        <source>GitHub</source>
+        <target>GitHub</target>
         <note>Context:
 File: admin-dev/themes/default/template/controllers/dashboard/helpers/view/view.tpl:122</note>
       </trans-unit>
       <trans-unit id="13df627c018625ce68ca2200a869dd65">
-        <source>Report issues in the Bug Tracker</source>
-        <target>Report issues in the Bug Tracker</target>
+        <source>Report issues on GitHub</source>
+        <target>Report issues on GitHub</target>
```
Comment (Quetzacoalt91) [inline app/Resources/translations/default/AdminDashboardFeature.xlf]: These changes are not needed, as they will be regenerated later.

Comment (MathiasReker) [reply app/Resources/translations/default/AdminDashboardFeature.xlf]: Fixed :)

**Comment by MathiasReker** [comment]
Can I do anything? :)

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
Thank you @MathiasReker

**Pull request merged**